### DISCUSSION TITLE
Remix里面无法正常显示测试币余额

### DISCUSSION THREAD

**Discussion by zedz** [IntensiveCoLearning/Web3-URL#129] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/931e3544-5d52-4f78-813e-8019523359d5)
钱包正常连接了，选择了自定义网络。
选择Injected Provider的时候，能正常显示网络，但是余额0。

![image](https://github.com/user-attachments/assets/663065c7-e3e1-4113-9a51-557aeb54b83e)
选择WalletConnect的时候，能正常显示余额，但连结钱包时无法选择自定义网络。

求解！

**Comment by qzhodl** [comment]
remix在哪里显示余额了？

**Reply by zedz** [reply]
左边红色框框标记出来了，account那里后面会显示余额

**Reply by luffythink** [reply]
你看看是不是没有及时刷新，我这边同样的步骤显示是正常的。

**Reply by zedz** [reply]
好像不行，重启浏览器，重启remix，重连钱包，都试了很多次。

**Comment by qizhou** [comment]
看左上角的钱包，选择的是`Injected Provider - OKX Wallet`。我猜想是OKX Wallet对其他网络支持不太好。

我个人测试用的MetaMask，余额显示是正常的。

<img width="240" alt="Screenshot 2024-07-21 at 10 59 34 PM" src="https://github.com/user-attachments/assets/2c464ba3-dda5-4ff9-9328-bf231e4f4946">

**Reply by zedz** [reply]
谢谢回复！可是okx和metamask我都试过了，都不行，也都尝试了很多次。

**Reply by qizhou** [reply]
其他测试网有类似的问题吗？

**Reply by zedz** [reply]
这是我第一次部署合约，没有其他测试币，不太清楚怎么验证？

**Reply by qizhou** [reply]
可以给我一个地址，我发一些sepolia的测试代币

**Reply by zedz** [reply]
0xf95bdaAA2391ECedfD5cbb32B51C86360b0654F2

谢谢！

**Comment by zedz** [comment]
前几天成功的部署了helloworld，刚刚又想试一下，还是无法获取钱包余额，怎么刷新都不行。还把上回的测试币弄丢了...

**Comment by zedz** [comment]
![image](https://github.com/user-attachments/assets/485b57cb-5d8f-477d-aefe-9aa68a5b9709)

测试币找到了
但是这个不显示余额到底咋整？

**Reply by qzhodl** [reply, marked as answer]
感觉像是网络问题？

**Reply by zedz** [reply]
换了浏览器 改了全局  终于可以了
